src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowShuntTest.java

```diff
@@ -430,6 +430,19 @@ void testIncrementalVoltageControl() {
         assertEquals(0, shunt.getSectionCount());
     }
 
+    @Test
+    void testIncrementalVoltageControlNotPlausibleTargetV() {
```

Comment:
why 'incremental and not just `testNotPlausibleTargetV()` ?

this feature is unrelated to incremental control